services/gateway/apis/http-version3/swagger/parameters/blockchainApps.json

```
@@ -24,19 +24,11 @@
         "format": "id",
         "minLength": 8
     },
-    "state": {
+    "blockchainAppState": {
         "name": "state",
         "in": "query",
-        "description": "State of the blockchain application.",
-        "type": "string",
-        "minLength": 1
-    },
-    "isDefault": {
-        "name": "isDefault",
-        "in": "query",
-        "description": "Filter default blockchain application.",
-        "type": "boolean",
-        "minLength": 1
+        "description": "State of the blockchain application (can be expressed as CSV. Accepted values: registered,active,terminated).",
```

Please test the change and adjust if the recommendation is incorrect.
```suggestion
        "description": "State of the blockchain application. Can be expressed as CSV.\nAccepted values: 'registered', 'active', 'terminated'.",
```

---

Applied.